packages/thirdweb/src/react/web/ui/ConnectWallet/Details.tsx

```diff
@@ -1083,12 +1089,32 @@ function InAppWalletUserInfo(props: {
         "../../../../wallets/in-app/web/lib/auth/index.js"
       );
 
+      const isInAppWallet =
+        adminWallet &&
```

Comment:
How does adminWallet mean it's an in-app wallet?

Reply:
not on its own, but it can be undefined so just checking that its defined here. Also adding it to the query